estimator/prob.py

```diff
@@ -2,6 +2,44 @@
 from sage.all import binomial, ZZ, log, ceil, RealField, oo, exp, pi
 from sage.all import RealDistribution, RR, sqrt, prod, erf
 from .nd import sigmaf
+from .conf import max_n_cache
+
+
+chisquared_table = {i: None for i in range(2*max_n_cache+1)}
+for i in range(2*max_n_cache+1):
+    chisquared_table[i] = RealDistribution('chisquared', i)
+
+
+def conditional_chi_squared(d1, d2, lt, l2):
```

Comment:
Can you document parameters and add some doctests?

Reply:
Done as of latest commit.